services/construction_service.go

```diff
@@ -88,22 +88,22 @@ func (s *ConstructionAPIService) ConstructionDerive(
 
 // estimateSize returns the estimated size of a transaction in vBytes.
 func (s *ConstructionAPIService) estimateSize(operations []*types.Operation) float64 {
-	size := bitcoin.TransactionOverhead
+	size := zen.TransactionOverhead
```

Comment:
Modify this function to calculate the size taking in consideration Replay protection etc ..
Introduce a test to verify correctness